src/game_battlealgorithm.cpp

```diff
@@ -1428,6 +1428,8 @@ bool Game_BattleAlgorithm::Skill::IsReflected() const {
 			if (party_target) {
 				std::vector<Game_Battler*> targetlist;
 				party_target->GetActiveBattlers(targetlist);
+				// Return false if no targets exist
+				if (targetlist.size() == 0) return false;
```

Comment:
Is this needed? The loop below will already not iterate anything if the list is empty

Reply:
You are right, this check is unnecessary.